src/components/Atoms/Combobox.tsx

```diff
@@ -100,7 +100,7 @@ export const Combobox = (props: ComboboxProps) => {
           }}
         >
           {props.aboveOptionsElement}
-          <Menu class="flex w-full flex-col space-y-1 overflow-y-auto bg-gray-800 shadow-lg ring-1 ring-black ring-opacity-5 overflow-x-hidden">
+          <Menu class="flex w-full rounded flex-col space-y-1 overflow-y-auto bg-pink-50 dark:bg-gray-800 shadow-lg ring-1 ring-black ring-opacity-5 overflow-x-hidden">
```

Comment:
```suggestion
          <Menu class="flex w-full flex-col space-y-1 overflow-y-auto rounded bg-pink-50 shadow-lg ring-1 ring-black ring-opacity-5 overflow-x-hidden dark:bg-gray-800">
```

Linter wants a different class order